### PR TITLE
refs #49203 Add CkEditor plugins

### PR DESCRIPTION
**Pull request by victor-aston**

### CODE REVIEW

**Review by boddy-devbranch — APPROVED** [review]

File: config/optional/linkit.linkit_profile.default.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+uuid: aa0dddc0-45e3-4ab3-8e99-eb5096981ad3
```
Comment (andrew-sereda) [inline config/optional/linkit.linkit_profile.default.yml]: Install configs should not contain uuid

File: config/optional/linkit.linkit_profile.default.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+_core:
+  default_config_hash: 0Jw_BFJCCtWk187tIYvME58VFpYwPaAdrc4eRtAyHH0
```
Comment (andrew-sereda) [inline config/optional/linkit.linkit_profile.default.yml]: This is also should be removed

**Review by andrew-sereda — CHANGES_REQUESTED** [review]

File: config/optional/linkit.linkit_profile.default.yml (hunk 1 above)
Comment (victor-aston) [reply config/optional/linkit.linkit_profile.default.yml]: done.

File: config/optional/linkit.linkit_profile.default.yml (hunk 2 above)
Comment (victor-aston) [reply config/optional/linkit.linkit_profile.default.yml]: done.

**Review by andrew-sereda — APPROVED** [review]

**Pull request merged**